src/extensions/askUserForConsent.ts

```diff
@@ -49,31 +46,19 @@ export async function retrieveRoleInfo(role: string) {
 }
 
 /**
- * Displays roles and corresponding descriptions and asks user for consent.
+ * Displays roles that will be granted to the extension instance and corresponding descriptions.
  * @param extensionName name of extension to install/update
  * @param projectId ID of user's project
  * @param roles roles that require user approval
  * @return {Promise<?>} returns promise
```

Comment:
not in this CL but maybe just annotate with the function with ` : Promise<void>` rather than putting it in a comment?